src/sbmlutils/report/sbmlinfo.py

```diff
@@ -156,6 +156,43 @@ def info_sbase(sbase: libsbml.SBase) -> Dict[str, Any]:
                     replaced_elements_combined = "".join(replaced_elements)
                 info["replaced_elements"] = replaced_elements_combined
 
+        # distrib
+        item_distib = sbase.getPlugin("distrib")
+        if item_distib and type(item_distib) == libsbml.DistribSBasePlugin:
+
+            # list to store uncertainities
+            info["uncertainties"] = []
+
+            listOfUncertainties = item_distib.getListOfUncertainties()
+            for uncertainty in listOfUncertainties:
+
+                u_dict = {
```

Comment:
The uncertainty is an SBase, so you have to get the sbase info first, i.e. the SBase dictionary. See similar code for the other objects.

Reply:
see `info_sbaseref`